packages/react-tooltip-trigger/src/common/useTooltipContext.ts

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import { TooltipContext } from './TooltipContext.types';
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export const internal__TooltipContext = React.createContext<TooltipContext>({
+  // These default values are replaced by TooltipProvider
+  Tooltip: () => null, // eslint-disable-line @typescript-eslint/naming-convention
+  manager: undefined,
+  portalRoot: document.body,
```

Comment:
- this will throw on SSR as `document` is not defined. As an option:

```ts
portalRoot: typeof document === 'undefined' ? null : document.body
```

- another problem, that we should consume `document` from `FluentProvider` to render into a proper `document`

Reply:
This is just the default that will be replaced later by TooltipProvider. I will use `undefined` here. And in the TooltipProvider, I'll use the document from `FluentProvider`.